### ISSUE TITLE
Quick info shows irrelevant warning instead of error

### ISSUE THREAD

**Issue by Neme12** [dotnet/roslyn#26022]
Insert this piece of code into VS:
```c#
class C
{
    void M()
    {
        int Function()
        {
            
        }
    }
}
```
I see a red squiggle on `Function` so I move the cursor over it and expect to see what the error is. Instead, I only get an unhelpful warning. This forces me to open the error list.

![image](https://user-images.githubusercontent.com/11444821/38469106-d0d25902-3b4f-11e8-92d6-1264b3d775cf.png)

It would be nice if this showed all diagnostics at that location including an icon:

![image](https://user-images.githubusercontent.com/11444821/38469235-9f5b7bfe-3b51-11e8-92c9-0f82df2aa590.png)

possibly even with the diagnostic IDs (just like in the lightbulb preview window):

![image](https://user-images.githubusercontent.com/11444821/38469306-f5d4ee74-3b52-11e8-9562-db549df74313.png)

**Comment by Neme12** [comment]
@jinujoseph Should this be moved?

**Comment by davkean** [comment]
@Neme12 This has been fixed in 15.8 Preview 3 - it was a regression in 15.7 which did show all errors. Note, that the icons/links suggestion was lost in the move internally. I recommend that you manually ask for this via Help -> Send Feedback -> Report a Problem so that others can upvote it and you can get keep track of the issue.

**Comment by Neme12** [comment]
@davkean Thanks, I will do that!
